README.md

```diff
@@ -134,7 +134,7 @@
   * [Archipel](http://archipelproject.org/) - Manage and supervise virtual machines using Libvirt.
   * [CloudStack](http://cloudstack.apache.org/) - Cloud computing software for creating, managing, and deploying infrastructure cloud services.
   * [Cobbler](http://cobbler.github.io/) - Cobbler is a Linux installation server that allows for rapid setup of network installation environments.
-  * [Cracow Cloud One - Polish Private Cloud](http://cc1.ifj.edu.pl/) - The CC1 system provides a complete solution for Private Cloud Computing.
+  * [Cracow Cloud One - Polish Private Cloud](http://cc1.ifj.edu.pl/pl/) - The CC1 system provides a complete solution for Private Cloud Computing.
```

Comment:
this is an english sysadmin list, so I doubt we want to link to a polish version of the site.